exchanges/exchange_test.go

```diff
@@ -2897,3 +2897,8 @@ func TestWebsocketSubmitOrder(t *testing.T) {
 	_, err := (&Base{}).WebsocketSubmitOrder(t.Context(), nil)
 	require.ErrorIs(t, err, common.ErrFunctionNotSupported)
 }
+
+func TestWebsocketSubmitOrders(t *testing.T) {
+	_, err := (&Base{}).WebsocketSubmitOrders(t.Context(), nil)
```

Comment:
We missed it, but this needed a `t.Parallel`.
I'll add it in another branch.
Mentioning it in case I'm missing why there wasn't one.